internal/config/plugin.go

```diff
@@ -71,7 +71,7 @@ type Plugin struct {
 	Info    *PluginInfo   // json file containing info
 	Srcs    []RuntimeFile // lua files
 	Loaded  bool
-	Default bool          // pre-installed plugin
+	Default bool // pre-installed plugin
```

Comment:
I have checked that `gofmt -l -w` does this. Probably because there is one line without a comment between them.  I remember `clang-format` having this option too: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#aligntrailingcomments